src/sender.js

```diff
@@ -101,7 +107,10 @@ async function main({ msg, ackMsg, nackMsg, sendToQueue }) {
         console.error(`Tx Failed for event Tx ${job.transactionReference}`)
         failedTx.push(job)
 
-        if (
+        if (e.message.includes('Insufficient funds')) {
+          console.error('Insufficient funds')
```

Comment:
Will the similar message be logged anyway in the line 106? Will we have duplicated lines in the logs?

Reply:
You are right. We can just remove this comment, or make it more clear. What about something like:

```Insufficient funds: ${currentBalance}. Stop consuming messages until the balance is ${minimumBalance}.```